README.md

```diff
@@ -286,7 +286,18 @@ Available launch-file options:
 
 ### Example 2: "Industrial Robots with only one interface" (Gazebo simulation)
 
-- **TBA**
+- Launch file: ros2_control_demo_bringup rrbot_system_position_only_gazebo.launch.py
```

Comment:
This is great! Thanks! Could you also reference "controllers.yaml" and URDF used in that example. See how this is done for other examples.

Reply:
@destogl I will continue on this PR after the #190 is merged because then the naming can be aligned for the joint_trajectory_position_controller.

Reply:
Other examples also do not mention the controllers yaml or urdf. i do not understand what you mean. The gazebo example also uses the rrbot_controllers.yaml file like the non-gazebo example

Reply:
Oh, I see now… you are working on foxy. Please make PR against master. Otherwise, we are correcting all the time some old code. We can then rebase this to foxy later.